### DISCUSSION TITLE
Numlock on MacBook during Horus installation

### DISCUSSION THREAD

**Discussion by garethstewart** [elementary/triage#169] [Q&A]
Hi, updating from 6.1 to 7 on pre 2008 MacBook, but have run into an issue - numlock appears to be active, as the only keys working on my keyboard the function keys, delete, and the keys used for a number pad when lock is on (m, ., and / are outputting 0, ., and =, j, k, and l are outputting 1, 2, and 3, etc.). There is no numlock key, and modifier keys (shift, control, alt, etc.) are not working.

Any ideas how I remove numlock, bearing in mind ONLY number pad keys work?

**Comment by sysfu** [comment]
Having the same issue since upgrading to 7x.

Tried setting activate-numlock=false in  /etc/lightdm/io.elementary.greeter.conf but did not resolve. 

Reference: 

https://elementaryos.stackexchange.com/questions/12452/getting-numlock-to-be-on-by-default-on-start-login-screen-solved/28048#28048

https://elementaryos.stackexchange.com/questions/373/how-do-i-enable-numlock-at-login/17105#17105
